tests/PKSim.Tests/ProjectConverter/v10/Converter9To10Specs.cs

```diff
@@ -108,5 +108,55 @@ public void should_have_updated_the_ontogeny()
          var enzyme = _individual.MoleculeByName<IndividualEnzyme>("E1_Intracellular_Endosomal");
          enzyme.Ontogeny.Name.ShouldBeEqualTo("CYP2C18");
       }
+
+      [Observation]
+      public void should_have_updated_the_fraction_expressed_intracellular_based_on_the_localization()
```

Comment:
@Yuri05 using your tests. All good. And this is also what is used in snapshot that's why it works

Reply:
cool :)